src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -11,7 +11,7 @@
   >
     <DataSource
       v-slot="{ data: traffic, error, refresh }: StatsSource"
-      :src="props.data.dataplaneType === 'standard' ? `/meshes/${route.params.mesh}/dataplanes/${route.params.dataPlane}/stats` : ''"
+      :src="props.data.dataplaneType === 'delegated' ? '' : `/meshes/${route.params.mesh}/dataplanes/${route.params.dataPlane}/stats/${props.data.dataplaneType === 'builtin' && props.data.dataplane.networking.gateway ? props.data.dataplane.networking.gateway.tags['kuma.io/service'] : 'localhost_'}`"
```

Comment:
**Comment**: Not in love with this, but I didn’t want to introduce a second source: it would be effectively duplicate the current stats source with the only difference being the hard-coded `localhost_` service prefix.

Reply:
At least short-term, this approach is what we should do.

There are some details though which you might like to tweak but I don't mind either way.

For sidecars the "name of the thing" is "localhost", for gateways its "service-name". The underscore is a little unfortunate, but thats just because its what the envoy line format seems to use to separate the port (but inconsistently between sidecars and gateways it seems). I wonder myself where the port comes into it with gateways.

Following that, other options could be:

- Look for `=== 'localhost'` in the data layer and add the `_`. Whilst that has the potential to clash, I was wondering the other night what happens if I make a `kuma.io/service: localhost_8888`, so I think that could have a different knock on elsewhere. 
- Give `DataPlane` or `DataplaneOverview` an `inboundName/Prefix` property (or similar) in the data layer and derive it there, then use `/meshes/${route.params.mesh}/dataplanes/${route.params.dataPlane}/stats/${route.params.dataPlane.inboundName}` instead.

Whatever you do it will have to happen where we want the DataSource to sync between multiple sources also.

Reply:
Looking at the below code, if you do the second option I mention in this comment, a lot of the conditionals can then depend on the existance of the `inboundName/Prefix` property. If thats empty, don't show the viz.

Reply:
> * Give `DataPlane` or `DataplaneOverview` an `inboundName/Prefix` property (or similar) in the data layer and derive it there, then use `/meshes/${route.params.mesh}/dataplanes/${route.params.dataPlane}/stats/${route.params.dataPlane.inboundName}` instead.

I might do this eventually, but I’m gonna wait and let the rest come together before tweaking this further. What I currently have should be correct and workable for the time being. Would be good to improve though.

Reply:
Sounds good 👍